simulation/src/range_likelihood.cpp

```diff
@@ -1,5 +1,9 @@
-#include <GL/glew.h>
 #include <ctime>
+#include <random>
+
+#include <boost/math/distributions/normal.hpp>
```

Comment:
Is this one still required? I assumed the whitelist for now was only OutOfCoreDiskContainer and metslib. Is this one related?

Reply:
Missed this in whitelist `costFunction5` uses `pdf`/`cdf` which don't have a direct C++14 alias.